src/ecalc/libraries/libecalc/common/libecalc/input/yaml_types/components/installation.py

```diff
@@ -1,16 +1,39 @@
+from typing import Any, Dict, List, Type
+
 from libecalc.dto.base import InstallationUserDefinedCategoryType
 from libecalc.expression import Expression
 from libecalc.input.yaml_types import YamlBase
 from libecalc.input.yaml_types.components.category import CategoryField
+from libecalc.input.yaml_types.components.legacy.yaml_fuel_consumer import (
+    YamlFuelConsumer,
+)
 from libecalc.input.yaml_types.placeholder_type import PlaceholderType
 from libecalc.input.yaml_types.temporal_model import TemporalModel
 from pydantic import Field
 
 
+def replace_installation_property_with_legacy_json_schema(schema: dict, property_key: str, property_ref: str) -> None:
```

Comment:
possible to have this "util function" as a helper to yaml for now instead of replicating in every "yaml module"?